android/assets/jsons/translations/Spanish.properties

```diff
@@ -818,22 +811,14 @@ Branches completed = Ramas completadas
 Undefeated civs = Naciones no derrotadas
  # The \n here means: put a newline (enter) here. If this is omitted, the sidebox in the diplomacy overview will become _really_ wide.
  # Feel free to replace it with a space and put it somewhere else in your translation
- # Requires translation!
-Turns until the next\ndiplomacy victory vote: [$turnsTillNextDiplomaticVote] = 
- # Requires translation!
-Choose a civ to vote for = 
- # Requires translation!
-Choose who should become the world leader and win a diplomatic victory! = 
- # Requires translation!
-Voted for = 
- # Requires translation!
-Vote for [civilizationName] = 
- # Requires translation!
-Continue = 
- # Requires translation!
-Abstained = 
- # Requires translation!
-Vote for World Leader = 
+Turns until the next\ndiplomacy victory vote: [$turnsTillNextDiplomaticVote] = Turnos hasta la siguiente\n votación de victoria diplomatica: [$turnsTillNextDiplomaticVote]
```

Comment:
elección? "diplomática"?

Reply:
What?

Reply:
Aaahh ya.
I'll ask over and see what others think.